### PR TITLE
Feature/703 minmax zoom for wms wmts

### LINKED ISSUE #703: Be able to set minZoom and maxZoom for WMSLayer & WMTSLayer
Today, it's only possible to set minZoom and maxZoom for Vector layers.
We should also add this possibility for WMSLayer & WMTSLayer.

Change needed in both client and admin.

### PR DESCRIPTION
**Pull request by jesade-vbg**
Closes #703

### CODE REVIEW

File: new-client/src/models/layers/WMSLayer.js [hunk 1]
```diff
@@ -37,6 +37,15 @@ class WMSLayer {
...
 
+    const minZoom =
+      config?.minZoom != null && config?.minZoom >= 0
+        ? config.minZoom
+        : undefined;
+    const maxZoom =
+      config?.maxZoom != null && config?.maxZoom >= 0
+        ? config.maxZoom
+        : undefined;
+
```
Comment (sweco-sesqqq) [inline new-client/src/models/layers/WMSLayer.js:48]: `const minZoom = config?.minZoom >= 0 ? config.minZoom : undefined` should be enough?

File: new-client/src/models/layers/WMTSLayer.js [hunk 2]
```diff
@@ -67,13 +67,24 @@ class WMTSLayer {
...
 
+    const minZoom =
+      config?.minZoom != null && config?.minZoom >= 0
+        ? config.minZoom
+        : undefined;
+    const maxZoom =
+      config?.maxZoom != null && config?.maxZoom >= 0
+        ? config.maxZoom
+        : undefined;
+
```
Comment (sweco-sesqqq) [inline new-client/src/models/layers/WMTSLayer.js:78]: See previous comment

File: new-client/src/plugins/Print/PrintModel.js [hunk 3]
```diff
@@ -398,6 +398,9 @@ export default class PrintModel {
     const size = this.map.getSize();
```
Comment (sweco-sesqqq) [inline new-client/src/plugins/Print/PrintModel.js:398]: This should be in a pr of it's own right? Or does your change affect the print?

**Review by sweco-sesqqq — COMMENTED** [review]
I've checked all files, see comments

File: new-client/src/models/layers/WMSLayer.js (hunk 1 above)
Comment (jesade-vbg) [reply new-client/src/models/layers/WMSLayer.js:48]: Correct. Will amend!

File: new-client/src/models/layers/WMTSLayer.js (hunk 2 above)
Comment (jesade-vbg) [reply new-client/src/models/layers/WMTSLayer.js:78]: Correct. Will amend!

File: new-client/src/plugins/Print/PrintModel.js (hunk 3 above)
Comment (jesade-vbg) [reply new-client/src/plugins/Print/PrintModel.js:398]: Oh... no... Yes, you are right. Good find. Commited by misstake.

**Comment by jesade-vbg** [comment]
Fixes pushed. Ok to merge?

**Review by jacobwod — APPROVED** [review]
We can give it a try.

**Pull request merged**